helm/hpcc/templates/_helpers.tpl

```diff
@@ -402,7 +402,7 @@ Pass in dict with root
 {{- $dllStorage := ($storage.dllStorage | default dict) -}}
 {{- if not $dllStorage.plane -}}
 {{- if (not (include "hpcc.hasPlaneForLabel" (dict "root" .root "label" "dll"))) }}
-{{ include "hpcc.addVolumeMount" (dict "root" .root "me" $dllStorage "name" ($dllStorage.plane | default "dllstorage") "path" "queries") }}
+{{ include "hpcc.addVolumeMount" (dict "root" .root "me" $dllStorage "name" "dllstorage" "path" "queries") }}
```

Comment:
this change (and next) are cleaning up code - since plane is always null inside the if